dev-libs/smooth/smooth-0.9.9.recipe

```diff
@@ -81,6 +81,7 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
+	export LDFLAGS="-liconv"
```

Comment:
Maybe this could be upstreamed here:

https://github.com/enzo1982/smooth/blob/master/Makefile#L169

or here?

https://github.com/enzo1982/smooth/blob/master/Makefile#L226